src/views/SurgeOverview/Readiness/EmergencyResponseUnitOwnerCard/index.tsx

```diff
@@ -0,0 +1,148 @@
+import { useCallback, useContext } from 'react';
+import { _cs } from '@togglecorp/fujs';
+import { generatePath } from 'react-router-dom';
+
+import List from '#components/List';
+import Link from '#components/Link';
+import Header from '#components/Header';
+import TextOutput from '#components/TextOutput';
+import Heading from '#components/Heading';
+import DateOutput from '#components/DateOutput';
+import { paths } from '#generated/types';
+import useTranslation from '#hooks/useTranslation';
+import RouteContext from '#contexts/route';
+import { resolveToString } from '#utils/translation';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+type GetERUOwners = paths['/api/v2/eru_owner/']['get'];
+type GetERUOwnersResponse = GetERUOwners['responses']['200']['content']['application/json'];
+type ERUOwnerListItem = NonNullable<GetERUOwnersResponse['results']>[number];
+
+interface Props {
+    className?: string;
+    data: ERUOwnerListItem;
+}
+const emergencyResponseUnitKeySelector = (item: ERUOwnerListItem['eru_set'][number]) => item.id;
```

Comment:
Move props declaration closer to component
```suggestion
const emergencyResponseUnitKeySelector = (item: ERUOwnerListItem['eru_set'][number]) => item.id;

interface Props {
    className?: string;
    data: ERUOwnerListItem;
}

```